server/Main.hs

```diff
@@ -102,7 +107,7 @@ server externs initNamesEnv initEnv port = do
               search = fst . TS.typeSearch (Just []) initEnv (P.emptyCheckState initEnv)
               results = nubBy ((==) `on` fst) $ do
                 elab <- elabs
-                let strictMatches = search (replaceTypeVariablesAndDesugar (\nm s -> P.Skolem P.NullSourceAnn nm s (P.SkolemScope 0)) elab)
+                let strictMatches = search (replaceTypeVariablesAndDesugar (\nm s -> P.Skolem P.NullSourceAnn nm Nothing s (P.SkolemScope 0)) elab)
```

Comment:
Skolems are now tagged with their kinds as per [the PolyKinds PR](https://github.com/purescript/purescript/pull/3779), so there's an additional argument:

https://github.com/purescript/purescript/blob/62c5862430f91a9249046487bd80bd058b3df915/lib/purescript-ast/src/Language/PureScript/Types.hs#L73

If setting this to `Nothing` is the wrong approach, I'd appreciate a note on where I might find this information.